Makefile

```diff
@@ -63,9 +63,11 @@ bump-primage:
 deploy:
 	@./scripts/kcversion.sh
 	kubectl create namespace reconciler --dry-run=client -o yaml | kubectl apply -f -
-	helm template reconciler --namespace reconciler --set "global.components={$(COMPONENTS)}" ./resources/reconciler > reconciler.yaml
+	git clone --single-branch --depth 1 https://github.com/kyma-project/control-plane.git
+	helm template reconciler --namespace reconciler --set "global.components={$(COMPONENTS)}" --set global.mothership_reconciler.enabled=true --set global.component_reconcilers.enabled=true --set compass.enabled=false --set mothership-reconciler.db.encryptionKey=5bc19d3a2032fb8795cd86e08b473a351631505d2522991266b1fb85f89bad5f --set mothership-reconciler.db.serviceHost=reconciler-postgresql --set mothership-reconciler.db.reconcilerUsername=postgres --set mothership-reconciler.db.reconcilerPassword=test --set postgresql.postgresqlPassword=test --set postgresql.persistence.enabled=false --set postgresql.initdbScriptsConfigMap=reconciler-postgresql-db-init ./control-plane/resources/kcp > reconciler.yaml
 	kubectl apply -f reconciler.yaml
 	rm reconciler.yaml
+	rm -rf control-plane
```

Comment:
cloning and removing everytime I want to deploy mothership.
What If I want to change somehting in chart?

Reply:
This is only for convenience when you are in reconciler repository and don't want to change to control-plane just to deploy the charts. If you need to test some changes in the chart, then you will in any case check out and work on control-plane as the original charts reside there.

Reply:
Is chart from control plane is ready to be deployed locally?

Reply:
Suggestion only:
If I understand it correctly this `make deploy` would be only a formal (old-school) way to deploy the reconciler, right? True `make deploy` will be under the `control-plane` repository, right? If yes - can we replace this logic with a simple log like `hello, I'm in the control-plane now`? if not - can we add a simple flag, env to specify way how we want to fetch control-plane chart? (git/local paht).

Reply:
I'm adding a makefile for reconciler to the control-plane repo so that we can deploy reconciler easily by `make deploy-reconciler`: https://github.com/kyma-project/control-plane/pull/1052

Reply:
I'm deleting the target from this repo completely to avoid code duplication.